mage_ai/io/qdrant.py

```diff
@@ -0,0 +1,136 @@
+        )
+
+    def create_collection(
+            self,
+            vector_size: int,
+            distance: models.Distance = None,
+            collection_name: str = None):
+        """
+            Create collection in qdrant db.
+            Args:
+                vector_size (int): dimension size of the vector.
+                distance (models.Distance): distance metric to use.
+                collection_name (str): name of the collection.
+                Defaults to the name defined in io_config.yaml.
+            Returns:
+                collection created.
+        """
+        collection_name = self.collection if collection_name is None else collection_name
+        distance = models.Distance.COSINE if distance is None else distance
+        return self.client.create_collection(
+            collection_name=collection_name,
+            vectors_config=models.VectorParams(
+                size=vector_size,
+                distance=distance),
+        )
+
+    def load(
+        self,
+        limit_results: int,
+        query_vector: List,
+        collection_name: str = None,
+        **kwargs,
+    ) -> DataFrame:
+        """
+            Loads the data from Qdrant with query_vector.
+            Args:
+                limit_results (int): Number of results to return.
+                query_vector (List): vector list used to query.
+                collection_name (str): name of the collection.
+                Defaults to the name defined in io_config.yaml.
+            Returns:
+                DataFrame: Data frame object loaded with data from qdrant
+        """
+        # Assume collection is already created and exists.
+        collection_name = self.collection if collection_name is None else collection_name
+
+        hitted_results = self.client.search(
+            collection_name=collection_name,
+            query_vector=query_vector,
+            limit=limit_results,
+            with_vectors=True,
+        )
+        self.client.close()
+
+        output_df = {}
+        output_df['id'] = [hit.id for hit in hitted_results]
+        output_df['payload'] = [hit.payload for hit in hitted_results]
+        output_df['score'] = [hit.score for hit in hitted_results]
+        output_df['vector'] = [hit.vector for hit in hitted_results]
+
+        return DataFrame.from_dict(output_df)
+
+    def export(
+        self,
+        df: DataFrame,
+        collection_name: str = None,
+        vector_size: int = None,
+        distance: models.Distance = None,
+        **kwargs,
+    ) -> None:
+        """
+            Save data into Qdrant.
+            Args:
+                df (DataFrame): Data to export.
+                collection_name (str): name of the collection.
+                vector_size (int): dimension size of vector.
+                distance (models.Distance): distance metric to use.
+        """
+        collection_name = self.collection if collection_name is None else collection_name
```

Comment:
same. this can be simplified to `collection_name = collection_name or self.collection`